### ISSUE TITLE
png2gas: adjust for openPMD 1.0.0

### ISSUE THREAD

**Issue by ax3l** [ComputationalRadiationPhysics/picongpu#1446]
The [png2gas](https://github.com/ComputationalRadiationPhysics/picongpu/tree/master/src/tools/png2gas) tools should be adjusted to write [`openPMD 1.0.0`](https://github.com/openPMD/openPMD-standard) files (and read those in PIConGPU examples).

**Comment by ax3l** [comment]
@n01r please be aware of this issue that `png2gas` might need *slight* modifications now

**Comment by n01r** [comment]
Thanks! So far it seemed that it would suffice for the needs since if the number of cells in each direction is equal to the pixel dimensions. Reading worked as intended and running worked as well.

**Comment by ax3l** [comment]
excellent. yes it does not really need the openPMD meta information yet, we read the data rather simple. feel free to report if you encounter any issues and make sure to use PNGwrite 0.6.0 due to the 8bit read bugs documented here: https://github.com/pngwriter/pngwriter/blob/0.6.0/CHANGELOG.md

**Comment by n01r** [comment]
Yep, had to build PNGwriter 0.6.0 since it was required and it was not available on the cluster.

**Comment by ax3l** [comment]
Already requested PNGwriter 0.6.0 to be installed a few hours ago, will take a few days due to vacations  but building your own is good!
